package.json

```diff
@@ -8,7 +8,9 @@
   "private": true,
   "workspaces": {
     "packages": [
-      "packages/*"
+      "packages/effection/pkg",
```

Comment:
Had to do this to be able to use the main effection package from `events`, not sure if this is correct or if there is a better solution?

Reply:
Yup, until we use yarn2 which _should_ resolve internal packages in dependency order, this is the only way I know. /cc @taras